prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% /6);
+    }
+    .card-deck {
+      padding: 3px;
+      flex-basis: calc(100% /6)
+    }
```

Comment:
This rule set has a selector for all elements with the `class="card-deck"` attribute, but we're only putting `id="card-deck"` on the parent and `class="card-wrapper"` on the children, so you can remove this rule set.